decidim-admin/app/controllers/decidim/admin/application_controller.rb

```diff
@@ -14,6 +14,9 @@ class ApplicationController < ::DecidimController
       include HttpCachingDisabler
       include DisableRedirectionToExternalHost
 
+      include RedesignLayout
+      redesign active: false
+
```

Comment:
@entantoencuanto could this be removed?

Reply:
The `redesign active: false` can be removed, but the concern has to be included